### PR TITLE
feature: implements manager of route

### LINKED ISSUE #15: 874ac634 - Create manager of route

### PR DESCRIPTION
**Pull request by cesar-ferreira**

### CODE REVIEW

File: movie_badgers/lib/core/routes/manager_route.dart [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:moviebadgers/modules/splash/view/page/splash_page.dart';
+
+class ManagerRoute {
+
+  static const SPLASH_PAGE = '/';
```
Comment (samirtf) [inline movie_badgers/lib/core/routes/manager_route.dart]: static const splashPage = '/';
https://dart.dev/guides/language/effective-dart/style

File: movie_badgers/lib/core/routes/manager_route.dart [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  static Route<dynamic> generateRoute(RouteSettings settings) {
+
+
+    switch( settings.name ) {
+
+      case SPLASH_PAGE :
```
Comment (samirtf) [inline movie_badgers/lib/core/routes/manager_route.dart]: splashPage
https://dart.dev/guides/language/effective-dart/style

**Review by samirtf — COMMENTED** [review]
We have to discuss about some lint rules.

File: movie_badgers/lib/core/routes/manager_route.dart (hunk 1 above)
Comment (cesar-ferreira) [reply movie_badgers/lib/core/routes/manager_route.dart]: Resolved

File: movie_badgers/lib/core/routes/manager_route.dart (hunk 2 above)
Comment (cesar-ferreira) [reply movie_badgers/lib/core/routes/manager_route.dart]: Resolved

**Review by samirtf — APPROVED** [review]
Thank you for your collaboration.

**Pull request merged**